android-artifacts.gradle

```diff
@@ -0,0 +1,22 @@
+uploadArchives {
+    /* task androidJavadocs(type: Javadoc) {
+       source = android.sourceSets.main.java.sourceFiles
+   }
+
+   task androidJavadocsJar(type: Jar, dependsOn: androidJavadocs) {
+       classifier = 'javadoc'
+       //basename = artifact_id
+       from androidJavadocs.destinationDir
+   }*/
+
+    task androidSourcesJar(type: Jar) {
+        classifier = 'sources'
+        //basename = artifact_id
+        from android.sourceSets.main.java.sourceFiles
+    }
+
+    artifacts {
+        archives androidSourcesJar
+        //archives androidJavadocsJar
```

Comment:
You're looking at the wrong repo BTW it's CoreUtils
On Mar 29, 2015 8:51 PM, "dylanrjames" <notifications@github.com> wrote:

> In android-artifacts.gradle
> <https://github.com/Raizlabs/RZAndroidBaseUtils/pull/6#discussion_r27361767>
> :
>
> > +
> > +   task androidJavadocsJar(type: Jar, dependsOn: androidJavadocs) {
> > +       classifier = 'javadoc'
> > +       //basename = artifact_id
> > +       from androidJavadocs.destinationDir
> > +   }*/
> > +
> > +    task androidSourcesJar(type: Jar) {
> > +        classifier = 'sources'
> > +        //basename = artifact_id
> > +        from android.sourceSets.main.java.sourceFiles
> > +    }
> > +
> > +    artifacts {
> > +        archives androidSourcesJar
> > +        //archives androidJavadocsJar
>
> Still not sure I'm understanding this - let's talk about this one in
> person.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Raizlabs/RZAndroidBaseUtils/pull/6/files#r27361767>.
>